### PR TITLE
Fix Unix restore for Microsoft.Net.Test.Sdk

### PR DESCRIPTION
**Pull request by ViktorHofer**
Lowercase package name as on Unix we are case sensitive. We are doing the same for other packages.

### CODE REVIEW

**Comment by AraHaan** [comment]
Nice, cant wait to merge this to my pull request too so that build can have a better chance at passing.

File: external/test-runtime/XUnit.Runtime.depproj [hunk 1]
```diff
@@ -19,7 +19,7 @@
     <XUnitAdapter Condition="'$(TargetGroup)' == 'netcoreapp'">xunit.runner.visualstudio.dotnetcore.testadapter</XUnitAdapter>
     <XUnitAdapter Condition="'$(TargetGroup)' == 'uap' OR '$(TargetGroup)' == 'uapaot'">xunit.runner.visualstudio.uwp.testadapter</XUnitAdapter>
     <XUnitAdapter Condition="'$(TargetGroup)' == 'netfx'">xunit.runner.visualstudio.testadapter</XUnitAdapter>
-    <MicrosoftNetTestSdkPackageName>Microsoft.NET.Test.Sdk</MicrosoftNetTestSdkPackageName>
+    <MicrosoftNetTestSdkPackageName>microsoft.net.test.sdk</MicrosoftNetTestSdkPackageName>
```
Comment (danmosemsft) [inline external/test-runtime/XUnit.Runtime.depproj]: For public packages (like `Microsoft.Diagnostics.Tracing.TraceEvent`) we are using PascalCasing. Why use lowercasing for internal packages?

Comment (danmosemsft) [reply external/test-runtime/XUnit.Runtime.depproj]: It seems on the face of it to be a recipe for more of htese issues

Comment (ViktorHofer) [reply external/test-runtime/XUnit.Runtime.depproj]: that's not related to internal vs public packages. We lower case the package name as the package folders are lowercased on Unix. Similar to https://github.com/dotnet/corefx/pull/33867/files/fe4aead92e8a08ba5fcde88dcd52ef1aeb2acdf1#diff-841e9e59d27acec708f2c73eaa3cf6a0R18 above. We use this property to access the folder in the filesystem below in a target.

Comment (ViktorHofer) [reply external/test-runtime/XUnit.Runtime.depproj]: We probably also use that property for the package name but the property's intent is to map to the package folder name.

Comment (danmosemsft) [reply external/test-runtime/XUnit.Runtime.depproj]: ah ok

**Comment by AraHaan** [comment]
Uh it seems on my pull request to still be causing MSB4019's even after merging this change? Mind investing my pull request?

**Pull request merged**